loggermeta/logger_meta.go

```diff
@@ -0,0 +1,45 @@
+// Package loggermeta stores and accesses the container struct in
+// context.Context.
+package loggermeta
+
+import (
+	"context"
+)
+
+// key is an unexported type for keys defined in this package. This prevents
+// collisions with keys defined in other packages.
+type key string
+
+// loggerMeta is the key for logger struct values in context.Context. Clients use
+// loggermeta.NewContext and loggermeta.FromContext instead of using this
+// key directly.
+var loggerMetaKey key = "loggerMeta"
+
+// LoggerMeta is a communication structure used to transport information in order
+// for a micro logger to use it when issuing logs.
+type LoggerMeta struct {
```

Comment:
I wanted type inside package match the package name. Otherwise I find that confusing. Also container has too much to do with docker :)

I tired LoggerContext but when you do `loggercontext.NewContext(ctx, loggercontext.New())` it also looks confusing. When I see `loggermeta.NewContext(ctx, loggermeta.New())`, I know immediately what is going on.